scprep/run/dyngen.py

```diff
@@ -0,0 +1,241 @@
+import pandas as pd
+from . import r_function
+
+_install = r_function.RFunction(
```

Comment:
Could you make this function into a generic `scprep.run.r_function.install_github`?